apps/fishing-map/public/locales/source/translations.json

```diff
@@ -256,6 +256,34 @@
     "uploadPublic": "Allow other users to see this dataset when you share a workspace",
     "uploadYourOwn": "Upload your own"
   },
+  "datasetUploadUI": {
```

Comment:
Everything here is related to ui. Is that adding any value? or why not just datasetUpload?